Messages/Messages/ChannelFacade.swift

```diff
@@ -51,3 +51,5 @@ struct ChannelFacade: ChannelServiceProtocol {
     }
     
 }
+
+
```

Comment:
If you don't need to change anything in this class, you shouldn't have touched it.